### DISCUSSION TITLE
FN2 Feedback

### DISCUSSION THREAD

**Discussion by AbbyANoble** [TeamBasedInquiryLearning/precalculus#117] [Show and tell]
Author: @AbbyANoble 
Reviewer: @tdegeorge 

https://teambasedinquirylearning.github.io/precalculus/2024/FN2.html

**Comment by tdegeorge** [comment]
Remark 2.2.1: Typo - the first sentence of the last paragraph (starting with "generally..."), there is an extra space before the period.

**Comment by tdegeorge** [comment]
Activity 2.2.9: The answer for part a has a typo. It should be x<5, not x<25.

Part b answer should just be 4 (not x<4).

**Comment by tdegeorge** [comment]
Activity 2.2.11: The numbers in the answer for part c and d look different - different font?

Also, just for consistency....when there are multiple answers, do we list the answer choices as well? Part c has multiple answers so that is why I am asking.

**Reply by AbbyANoble** [reply]
Looks like I was too lazy for math mode. 😆 

I added in the answer choice letters as well.

**Comment by tdegeorge** [comment]
Activity 2.2.13: Need to delete part c - I think it was copied from the previous activity

**Comment by AbbyANoble** [comment]
These changes are made in #134!
